2009/serial/B/B.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# Get the current directory so you can invoke it from anywhere
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+INPUTS=$DIR/../../misc/inputs
+
+$DIR/dna $INPUTS/B.in $INPUTS/B-query.in problem_output
```

Comment:
You could add a command to unzip the [B.zip](2009/misc/inputs/B.zip) file before the running. We can name zipped inputs as X.in.zip, where X is the problem name.

Reply:
@648trindade What do you think about this strategy for input names?

Reply:
I actually don't like this approach of passing a file to the program, instead of standard input.

I will do some rework to merge these 2 input files into a single one and make the program read from stdin